sdks/rust/src/alpha.rs

```diff
@@ -125,3 +125,115 @@ impl Alpha {
             .map(|pl| pl.into_inner().list)?)
     }
 }
+
+
+#[cfg(test)]
+mod tests {
+    use tokio;
+
+    // MockAlpha simulates Alpha's async methods for unit testing
+    struct MockAlpha {
+        capacity: i64,
```

Comment:
This is player tracking yeah? We don't have to remove it, but it'll eventually be deleted.

Reply:
Yep, it's for the payer tracking